src/cascade/input_data/db/csmr.py

```diff
@@ -0,0 +1,136 @@
+"""Upload csmr data to t3 table so EpiViz can access it for plotting."""
+
+import logging
+
+try:
+    from db_queries import get_outputs
+except ImportError:
+
+    class DummyGetOutputs:
+        def __getattr__(self, name):
+            raise ImportError(f"Required package db_queries not found")
+
+    get_outputs = DummyGetOutputs()
+
+
+from cascade.core.db import cursor
+
+
+CODELOG = logging.getLogger(__name__)
+
+
+def _csmr_in_t3(execution_context):
+    """Checks if data for the current model_version_id already exists in tier 3.
+    """
+
+    model_version_id = execution_context.parameters.model_version_id
+
+    query = """
+    SELECT exists(
+             SELECT * FROM epi.t3_model_version_csmr
+             WHERE model_version_id = %(model_version_id)s
+    )
+    """
+    with cursor(execution_context) as c:
+        c.execute(query, args={"model_version_id": model_version_id})
+        exists = c.fetchone()[0]
+
+    return exists == 1
+
+
+def _get_csmr_data(execution_context):
+
+    model_version_id = execution_context.parameters.model_version_id
+    add_csmr_cause = execution_context.parameters.add_csmr_cause
+    parent_loc = execution_context.parameters.drill
+
+    csmr_measure_id_deaths = 1  # measure type = deaths (1)
```

Comment:
Let's move all three of these to constants in the `__init__.py` of this package. There's the beginning of a mapping from measure names to measure ids in bundle.py which should probably also move into the top level. Eventually we'll want something better because having these kinds of magic constants is super dangerous but I'm doing it in my code too because I haven't thought of a better pattern yet.

Reply:
How do you mean this?  Something other than this, I guess.

(20180722) [nhawkins@cn497 db]$ more __init__.py 
MODEL_VERSION_ID = execution_context.parameters.model_version_id
CAUSE_ID = execution_context.parameters.add_csmr_cause
PARENT_LOC = execution_context.parameters.location_id

Reply:
I'm talking about csmr_measure_id_deaths, csmr_metric_id_rate and csmr_gbd_round_id. Those are constants which are defined by the external database but which we need to know here and likely elsewhere as well so they should be in a more central place. I suggest `cascade.input_data.db`

In the long term we should have mappings for all measure_ids (that we care about) and all metric_ids (that we care about). We'll never be working on more than one round at a time (I hope).

Did you look at the measure_id mapping in bundle.py? That's one possible way of approaching the more general version of this. I suggest you take that, move it to `cascade.input_data.db` and add the death measure to it then create a parallel thing for the metric_ids and just a regular constant for the round_id.